components/backends/sglang/slurm_jobs/scripts/benchmark_utils.sh

```
@@ -50,24 +50,30 @@ warmup_model() {
+    chosen_concurrencies=(1 2 4 8 16 32 64 128)
 
-    command=(
-        python3 -m sglang.bench_serving
-        --base-url "http://${service_host}:${service_port}"
-        --model ${served_model_name} --tokenizer ${model_path}
-        --backend sglang-oai
-        --dataset-name random --random-input ${isl} --random-output ${osl}
-        --random-range-ratio 1
-        --num-prompts ${num_prompts} --request-rate ${request_rate} --max-concurrency ${concurrency}
-    )
+	for concurrency in ${chosen_concurrencies[@]}
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)